SparkPost/2web2ui
# SA-1493: Always show view headers button

### What Changed
 - Removed logic that conditionally rendered view headers button.

### How To Test
 - Check that the view headers button in `/inbox-placement/details/{id}/mailbox-provider/{filter}` always exist.
 - You can check `/inbox-placement/details/143/sending-ip/147.253.223.88` on spc/108 for `sparkpost360@rackspace.sparkpostseed.com`
